pandas-stubs/core/groupby/generic.pyi

```diff
@@ -61,7 +61,7 @@ class SeriesGroupBy(GroupBy, Generic[S1]):
     def agg(self, func: list[AggFuncTypeBase], *args, **kwargs) -> DataFrame: ...
     @overload
     def agg(self, func: AggFuncTypeBase, *args, **kwargs) -> Series: ...
-    def transform(self, func, *args, **kwargs): ...
+    def transform(self, func: Callable, *args, **kwargs) -> Series: ...
```

Comment:
Looks like it needs to be `Callable | str` (in both of them)

Can you also create an issue in the pandas repo saying that the docs for `transform` need to indicate that the function parameter in `groupby.transform()` can be a string function name, and that an example should be included?

Reply:
yes I'll do that